### DISCUSSION TITLE
CCCV Charging

### DISCUSSION THREAD

**Discussion by Jadhav-Sourabh** [pybamm-team/PyBaMM#2200] [General]
We can easily implement CCCV charging in pybamm but in liionpack why this small thing is getting difficult. Please help

**Comment by TomTranter** [comment]
The solvers are implemented in a different way. It's on the to do list "in my head". If you are interested in helping make this work it would be welcome

**Comment by Jadhav-Sourabh** [comment]
I am interested

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by TomTranter** [reply]
Ok lets move this discussion to liionpack and work out a solution. Hopefully @martinjrobins and @tinosulzer may also have some suggestions

**Reply by Jadhav-Sourabh** [reply]
Yes, 
in liionpack, we need to specify some time period for each experiment expression.
pybamm is easy for experimentation. I got very good results for single cell model in pybamm. but for whole battery pack, I'm getting difficulty while preparing experiment function. How can I make such experiment expression so that I will get CCCV for pack?

**Reply by TomTranter** [reply]
It's just not possible right now. liionpack only accepts current and strips this out of the experimental protocol for CC and drive cycles. It will need development for what you want to do

**Reply by Jadhav-Sourabh** [reply]
Okay, let me know
how can I help you?
sourabh.jadhav@outlook.com
